app/src/org/commcare/android/database/user/models/FormRecord.java

```diff
@@ -69,6 +69,12 @@
     @Persisting(6)
     @MetaField(META_LAST_MODIFIED)
     private Date lastModified;
+
+    /**
+     * Unique id of the CC app to which this form belongs
+     */
+    @Persisting(7)

```

Comment:
Ah, then maybe I was trying to prevent a non-existent problem. This is being used in StorageUtils.filterByCurrentApp() to only return unsent forms that are part of the current app. When you have a second can you confirm that that seems unnecessary, and I'll remove it?